lib/livebook_web/live/session_live/persistence_live.ex

```diff
@@ -116,11 +116,17 @@ defmodule LivebookWeb.SessionLive.PersistenceLive do
                     tabindex="-1">
                     Cancel
                   </button>
-                  <button class="button-base button-blue"
-                    phx-click="confirm_file"
-                    tabindex="-1">
-                    Choose
-                  </button>
+                  <%= unless String.at(@draft_file.path, -1) == "/" do %>
```

Comment:
Instead of conditional you can set the `disabled` attribute, this should give the expected styling too. Also you can do the check with `FileSystem.File.dir?` :)